packages/nimbus/src/components/money-input/money-input.tsx

```diff
@@ -173,12 +164,11 @@ export const MoneyInputComponent = (props: MoneyInputProps) => {
   });
   const [styleProps, remainingProps] = extractStyleProps(recipeRestProps);
 
-  // Helper functions
-  const getAmountInputId = () => `${id}-amount`;
-  const getCurrencyDropdownId = () => `${id}-currency`;
-  const getAmountInputName = () => (name ? `${name}.amount` : undefined);
-  const getCurrencyDropdownName = () =>
-    name ? `${name}.currencyCode` : undefined;
+  // Id's and Names for each group input type
+  const amountInputId = getMoneyGroupAttribute(groupId, "amount");
+  const currencySelectId = getMoneyGroupAttribute(groupId, "currencyCode");
+  const amountInputName = getMoneyGroupAttribute(name, "amount");
+  const currencySelectName = getMoneyGroupAttribute(name, "currencyCode");
```

Comment:
Why have 4 functions you call all over the place when you can have one function you call 4 times?